packages/llama/src/chain-ids.ts

```diff
@@ -0,0 +1,3 @@
+export const OPTIMISM_CHAIN_ID = 10
```

Comment:
Would be preferable to use the Chains enum from the utils package

`import { Chains } from '@rabbitholegg/questdk-plugin-utils'`

Reply:
I couldn't get this to work :/. I've been fighting build issues this whole PR and wasn't feeling like working through it 🙃

Reply:
Try just building the utils package.

`pnpm build --filter=@rabbitholegg/questdk-plugin-utils`